docs/TunGo/src/pages/index.module.css

```
@@ -3,21 +3,78 @@
+  margin: 0 0 1rem;
 }
 
 .buttons {
   display: flex;
   align-items: center;
+  justify-content: flex-start;
+  gap: 0.65rem;
+  flex-wrap: wrap;
+}
+
+.buttons :global(.button) {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)